packages/quantic/cypress/integration/quantic-facet.cypress.ts

```diff
@@ -6,16 +7,49 @@ describe('quantic-facet', () => {
     cy.visit(`${Cypress.env('examplesUrl')}/s/quantic-facet`);
   });
 
-  it('should display the right label and content', () => {
-    cy.then(() => configure({field: 'language', label: 'Language'}))
+  it('should render correctly', () => {
+    configure({
+      field: 'objecttype',
+      label: 'Type',
+      numberOfValues: 8,
+      sortCriteria: 'automatic',
+      noSearch: false,
+    })
+      .get(facet.title)
+      .should('contain', 'Type')
+      .get(facet.collapse)
+      .should('exist')
+      .get(facet.searchbox)
+      .should('exist')
+      .get(facet.value)
+      .should('have.length', 8)
+      .get(facet.more)
+      .should('exist');
+  });
+
+  it('should render correctly with other field, less values, and no search', () => {
```

Comment:
Is there a reason why those 3 condition are listed ?
Do you think we should instead write the expected outcome like `display no searchbox` or do you think that that would be too specific ?

Reply:
I wrote the title this way trying to explain the use case, but that's more confusing than anything. As discussed offline, I'll reformat the tests to match the structure used in Atomic.